pallets/ddc-customers/src/lib.rs

```diff
@@ -840,7 +840,7 @@ pub mod pallet {
 			Ok(())
 		}
 
-		fn deposit_extra(owner: T::AccountId, amount: u128) -> Result<(), DispatchError> {
+		fn v	(owner: T::AccountId, amount: u128) -> Result<(), DispatchError> {
```

Comment:
Looks like an accidental renaming? Was it on purpose?

Reply:
yup